### ISSUE TITLE
straight-server build fails with "uninitialized constant BTC::ScriptChunk::OP_PUSHDATA4"

### ISSUE THREAD

**Issue by AlexanderPavlenko** [oleganza/btcruby#5]
https://travis-ci.org/MyceliumGear/straight-server/builds/76432132

```
/home/travis/.rvm/gems/ruby-2.2.1/gems/btcruby-1.1.4/lib/btcruby/script/script_chunk.rb:150:in `with_data': uninitialized constant BTC::ScriptChunk::OP_PUSHDATA4 (NameError)
    from /home/travis/.rvm/gems/ruby-2.2.1/gems/btcruby-1.1.4/lib/btcruby/script/script.rb:31:in `initialize'
    from /home/travis/.rvm/gems/ruby-2.2.1/gems/btcruby-1.1.4/lib/btcruby/block.rb:33:in `new'
    from /home/travis/.rvm/gems/ruby-2.2.1/gems/btcruby-1.1.4/lib/btcruby/block.rb:33:in `genesis_mainnet'
    from /home/travis/.rvm/gems/ruby-2.2.1/gems/btcruby-1.1.4/lib/btcruby/network.rb:35:in `mainnet'
```

Is something wrong on our side?

**Comment by oleganza** [comment]
Oops, it's a bug in name scopes. Fixing...

**Comment by oleganza** [comment]
@AlexanderPavlenko should be fixed in 1.1.5, please check.
